### ISSUE TITLE
Nested Tensor of dimension 0

### ISSUE THREAD

**Issue by anjali411** [pytorch/nestedtensor#24]
torch.tensor supports creation of a zero dimensional tensor
so 
```
torch.tensor([])
torch.tensor([[], []])
```
works.

However
```
nestedtensor.nested_tensor([[], []])
```
gives a RuntimeError. can we add zero dim support to Nested Tensors as well?

**Comment by nairbv** [comment]
@zou3519

**Comment by cpuhrsch** [comment]
Yes, I'll add this.

**Comment by cpuhrsch** [comment]
Supported via https://github.com/pytorch/nestedtensor/pull/28
